content/post/rxdnssd.md

```diff
@@ -217,8 +217,8 @@ I don't use `subscribeOn`, because DNSSD will create its own thread and it's not
 **Resolve** 
 -------------
 
-Next operation is `resolve`. I use this operation to resolve a service name to a target host, port number, and a txt record. 
-Unlike `browse` operation this one will return [`Transformer`](https://github.com/ReactiveX/RxJava/wiki/Implementing-Your-Own-Operators), that transforms an `Observable` of `BonjourService` to an `Observable` of resolved `BonjourService`. 
+Next operation is `resolve`. I use this operation to resolve target host service name, port number, and a txt record. 
+Unlike `browse` operation this one will return [`Transformer`](https://github.com/ReactiveX/RxJava/wiki/Implementing-Your-Own-Operators), that transforms an `Observable` of `BonjourService` into an `Observable` of resolved `BonjourService`. 
```

Comment:
Yes, actually only `browse` operation creates BonjourService object. All next operations, such as `resolve` and `queryRecords`,  just fill in properties of this object.